services/sidecar/rollup/services/resolver/resolver.go

```diff
@@ -0,0 +1,17 @@
+package resolver
```

Comment:
is this file unfinished or just a template?

Reply:
yes, so I started drafting this, but it shouldn't end up in the commit, must have accidentally add when fixing comments to the PR. will remove. thanks for spotting!